src/frontend/src/containers/tables/paymentmodule/PaymentsTable/PaymentsTableRow.tsx

```diff
@@ -99,6 +93,17 @@ export function PaymentsTableRow({
     );
   };
 
+  const renderPaymentStatusBox = (): React.ReactElement => {
```

Comment:
Shouldn't we display all possible statuses?

Reply:
yes WE should.

Reply:
fixed:
<img width="356" alt="image" src="https://github.com/user-attachments/assets/1704e8cf-de47-48d2-b7be-87b2f3ce7e74">